platform/api.io/src/org/netbeans/api/io/IOProvider.java

```diff
@@ -214,7 +214,8 @@ public abstract InputOutput getIO(@NonNull String name, boolean newIO,
      *
      * @param <IO>
      * @param <OW>
-     * @param <POS>
+     * @param <P>
+     * @param <F>
```

Comment:
tags without doc are useless. Either they get a description or they can be all removed.

Reply:
fixed below